bindings/java/gradlew

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
+
+#
+# Copyright 2015 the original author or authors.
```

Comment:
@GMNGeoffrey will this license header trigger any errors in our infrastructure? Maybe the files should live under a `third_party/` directory?

For a bit of context on why these files are useful to check in, see https://stackoverflow.com/a/40981755

Reply:
They won't because I got annoyed and turned off these checks (the snr was terrible). That said, we should check third party code in only under a third_party/ directory

Reply:
Also I can't comment on the binary file because GitHub, but is checking in a jar necessary? I know git doesn't like binary files (although that one is pretty small)

Reply:
Checking in the jar or the other gradle files isn't necessary, I just included them because that seems to be the standard that many other projects follow (more info in the stackoverflow link Scott included).  I can remove them if you would prefer, then anyone who wants to use the java api will just need to generate the files themselves

Reply:
This stuff should still go under `third_party/`. It's third party code we didn't write and has a different copyright

Reply:
Just sent out https://github.com/google/iree/pull/4580 to remove these files

Reply:
Thanks :-) I think we should probably also put the jar there? Unless the jar is actually are code? I can't tell since it's, well, a binary :grin: